### PR TITLE
feat: bump tofnd to 1.0.1

### PR DESCRIPTION
**Pull request by StephenFluin**

### CODE REVIEW

File: src/pages/validator/setup/manual.mdx [hunk 1]
```diff
@@ -18,15 +18,15 @@ Check the appropriate version for the network accordingly:
 <tab-item title="Mainnet">
  ```bash
  AXELARD_RELEASE=v0.35.5
```
Comment (ffe9f8) [inline src/pages/validator/setup/manual.mdx:20]:
```suggestion
 export AXELARD_RELEASE=v0.35.5
```

File: src/pages/validator/setup/manual.mdx [hunk 2]
```diff
@@ -18,15 +18,15 @@ Check the appropriate version for the network accordingly:
...
-TOFND_RELEASE=v0.10.1
+TOFND_RELEASE=v1.0.0
```
Comment (ffe9f8) [inline src/pages/validator/setup/manual.mdx:21]:
```suggestion
export TOFND_RELEASE=v1.0.0
```

File: src/pages/validator/setup/manual.mdx [hunk 3]
```diff
@@ -18,15 +18,15 @@ Check the appropriate version for the network accordingly:
...
 WASMVM_RELEASE=v1.3.1
 ```
 
 </tab-item>
 <tab-item title="Testnet">
  ```bash
  AXELARD_RELEASE=v0.35.5
-TOFND_RELEASE=v0.10.1
+TOFND_RELEASE=v1.0.0
 WASMVM_RELEASE=v1.3.1
```
Comment (ffe9f8) [inline src/pages/validator/setup/manual.mdx:30]:
```suggestion
export WASMVM_RELEASE=v1.3.1
```

File: src/pages/validator/setup/manual.mdx (hunk 1 above)
Comment (ffe9f8) [reply src/pages/validator/setup/manual.mdx:20]: etc.

**Comment by StephenFluin** [comment]
@ffe9f8 I don't think we should add the export keyword to the validator docs I touched which use tofnd because they weren't written to rely on it and it would make them inconsistent (our amplifier docs only rely on it if you're using the script to automate.
> Shell variables (defined without export) are like local variables that we can access only within that shell. Environment variables (defined with export) are like global variables that we can access within the defining shell and all its child shells, processes, and commands

**Review by ffe9f8 — APPROVED** [review]

**Comment by ffe9f8** [comment]
> @ffe9f8 I don't think we should add the export keyword to the validator docs I touched which use tofnd because they weren't written to rely on it and it would make them inconsistent (our amplifier docs only rely on it if you're using the script to automate.
> 
> > Shell variables (defined without export) are like local variables that we can access only within that shell. Environment variables (defined with export) are like global variables that we can access within the defining shell and all its child shells, processes, and commands

SGTM

**Pull request merged**